autogen.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
-if glibtoolize --version &> /dev/null
+glibtoolize=$(which glibtoolize)
```

Comment:
Not the case in my machine (I get an error message in stderr when glibtoolize is not found)